Content.Shared/Blocking/BlockingSystem.cs

```diff
@@ -40,10 +42,32 @@ public override void Initialize()
 
         SubscribeLocalEvent<BlockingComponent, GetItemActionsEvent>(OnGetActions);
         SubscribeLocalEvent<BlockingComponent, ToggleActionEvent>(OnToggleAction);
+        SubscribeLocalEvent<BlockingComponent, DestructionEventArgs>(OnBreak);
 
         SubscribeLocalEvent<BlockingComponent, ComponentShutdown>(OnShutdown);
     }
 
+    private void OnBreak(EntityUid uid, BlockingComponent component, DestructionEventArgs args)
+    {
+        if (_containerSystem.IsEntityInContainer(uid))
+        {
+            if (component.User == null)
+                return;
+
+            if (!EntityManager.TryGetComponent(component.User, out SharedHandsComponent? hands))
+                return;
+
+            foreach(var hand in hands.Hands)
+            {
+                if (hand.Value.HeldEntity == uid)
+                {
+                    _handsSystem.TryDrop((EntityUid) component.User, hand.Value, null, true, true, hands);
+                    return;
+                }
+            }
+        }
+    }
```

Comment:
this is definitely not the correct solution and won't work for anything else getting destroyed. it probably shouldn't be getting fixed in this PR regardless (as it seems to be a general bug with the stripping menu)